pkg/http/api_test.go

```diff
@@ -2477,20 +2402,17 @@ func TestAPIToken(t *testing.T) {
 		apiPath := "v1.0/invoke/fakeDaprID/method/fakeMethod"
 		fakeData := []byte("fakeData")
 
-		fakeReq := invokev1.NewInvokeMethodRequest("fakeMethod")
```

Comment:
do we need to add separate tests for the scenarios that are there in these fakeReq assignments? There are multiple being removed. Are we covering all the cases that are needed?

Reply:
All these objects that were removed where completely unused. My guess is that whoever wrote the test cases copy/pasted from cases where the reqs were needed, and didn't then realize they were unused.